config/internal/apiserver/deployment.yaml.tmpl

```diff
@@ -18,6 +18,54 @@ spec:
         component: data-science-pipelines
     spec:
       containers:
+        {{ if .APIServer.EnableRoute }}
+        - name: oauth-proxy
+          args:
+            - --https-address=:8443
+            - --provider=openshift
+            - --openshift-service-account=ds-pipeline-{{.Name}}
+            - --upstream=http://localhost:8888
+            - --tls-cert=/etc/tls/private/tls.crt
+            - --tls-key=/etc/tls/private/tls.key
+            - --cookie-secret=SECRET
```

Comment:
Is `--cookie-secret=SECRET` right? Almost looks like a bad copy paste from an example. But it's used in [this example](https://github.com/openshift/oauth-proxy/blob/master/contrib/sidecar.yaml#L65) too. A [different page](https://oauth2-proxy.github.io/oauth2-proxy/docs/configuration/overview#generating-a-cookie-secret) suggests it should be "the seed string for secure cookies (optionally base64 encoded)"

Reply:
you can do a followup for this